
exports.Register = async (req, res) => {
    try {
        
    } catch(err) {
        console.error(`Some error occures: ${err}`)
    }
};
exports.Users = async () => {
    console.log("users are here");
};